flowr/examples/gcd/DESCRIPTION.md

```
@@ -0,0 +1,26 @@
+Description
+===
+Computes the Greatest Common Divisor of two numbers using Euclid's
+subtraction-based algorithm, implemented entirely in dataflow using
+only flowstdlib functions.
+
+The algorithm: repeatedly subtract the smaller from the larger until
+they are equal. The equal value is the GCD.
+
+Root Diagram
+===
+<a href="root.dot.svg" target="_blank"><img src="root.dot.svg"></a>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)